AudioManagerUtilities/include/CAmDltWrapper.h

```diff
@@ -29,68 +29,68 @@
 #include "audiomanagertypes.h"
 
 #ifdef WITH_DLT
-	#include <dlt.h>
+    #include <dlt.h>
```

Comment:
I see this is merged already but **kpatel** please don't modify whitespace and code in the same commit. It makes it really hard for others to track the actual change.

(Also, why switch to spaces in only this file when about half of the other files use tabs?   Actually I found many that are mixed - so I'll file a separate ticket on cleaning up whitespace.)